core/src/com/unciv/models/stats/Stats.kt

```diff
@@ -146,7 +146,7 @@ open class Stats(
     /** **Mutating function**
      * Apply weighting for Production Ranking */
     fun applyRankingWeights() {
-        food *= 14
+        //food *= 14  // Do weighting in rankStatsForCityWork() instead
```

Comment:
Is that the only place we use this ranking function?

Reply:
Yup

Reply:
Put back in and derive it in the code instead